### ISSUE TITLE
Something of Interest

### ISSUE THREAD

**Issue by justinGilmer** [livecomsjournal/article_templates#29]
Just a really neat repository for converting LaTeX docs into pretty impressive looking HTML pages.

Not entirely useful for this journal, but could be nice to keep in mind.

https://github.com/arxiv-vanity/engrafo
Could also generate HTML pages in addition to pdf.

**Comment by davidlmobley** [comment]
This could be handy for authors who want to make websites in addition. 

Not sure what we should do with this info though; I'm not convinced the issue tracker is the right place for it. Maybe we should have a markdown file here which provides links to other tools which might be of use? Thoughts?

**Comment by dwsideriusNIST** [comment]
In the long run, it would be a good idea to have full-text HTML available somehow. My reason being that scrapers fail spectacularly on PDF documents.

**Comment by HaoZeke** [comment]
Well, it might be better to use [pandoc](pandoc.org) or even [org-mode exporter](https://orgmode.org/manual/HTML-export.html) with an appropriate set of stylesheets, mostly because they allow easier fine tuned control without resorting to using XML.

**Comment by davidlmobley** [comment]
I've heard pandoc mentioned a number of times; have you used it, @HaoZeke ?

**Comment by HaoZeke** [comment]
Yup @davidlmobley. In fact I've used it as the basis of many documentation setups, including [zenYoda](https://github.com/HaoZeke/zenYoda_Starter), [docuYoda](https://github.com/HaoZeke/docuYoda_Starter) and a [site with proper math formatting](https://github.com/HaoZeke/grimoire). For seeing how a document might be structured in markdown you can check [this report](https://github.com/HaoZeke/gasSweetening-Report/tree/master/report/md) and it's [pdf output](https://github.com/HaoZeke/gasSweetening-Report/blob/master/report/pdf/finalProj.pdf).

**Comment by davidlmobley** [comment]
Thanks, @HaoZeke !!
